docs/source/usage.rst

```diff
@@ -204,39 +214,44 @@ HTML files that can be used any HTTP server.
 Under some circumstances (ex Read The Docs) you might want to trigger the
 pydoctor API docs build together with the Sphinx build.
 
-This can be done by using the `pydoctor.sphinx_ext.build_apidocs` extension.
+This can be done by using the :py:mod:`pydoctor.sphinx_ext.build_apidocs` extension.
 
 Inside your Sphinx `conf.py` file enable and configure the extension in this
 way.::
 
     extensions.append("pydoctor.sphinx_ext.build_apidocs)
 
     pydoctor_args = [
-        '--quiet',
         '--project-name=YOUR-PROJECT-NAME',
         '--project-url=YOUR-PROJECT-HOME-URL',
         '--docformat=epytext',
         '--intersphinx=https://docs.python.org/3/objects.inv',
-        '--make-html',
         '--html-viewsource-base=https://github.com/ORG/REPO/tree/default',
         '--html-output={outdir}/api',
-        '--project-base-dir=/absolute/path/to/your/project',
-        '/absolute/path/to/your/project/package1'
+        '--project-base-dir=path/to/source/code',
+        'path/to/source/code/package1'
         ]
 
-The `{outdir}` will replaced with the Sphinx build dir.
+    pydoctor_url_path = '/en/{rtd_version}/api/'
+
+You can pass almost any argument to `pydoctor_args`
+in the same way you call `pydoctor` from the command line.
 
-You can pass any argument, in the same way you call `pydoctor` from the
-command line.
+You should not pass the `--make-html`, `--make-intersphinx` or `--quiet`
+arguments as this can break the build.
+The extension will add them automatically.
```

Comment:
What about?

```suggestion
You don't need to pass the `--make-html`, `--make-intersphinx` or `--quiet`
arguments.
The extension will add them automatically.
```

Reply:
That's fine, but you'd also have to remove `--make-intersphinx` from the list then.

By the way, I checked the code and `--make-intersphinx` does nothing if `--make-html` is passed as well, so it is harmless.